src/org/chartjs/HorizontalBarDAOChartView.js

```diff
@@ -0,0 +1,138 @@
+foam.CLASS({
+  package: 'org.chartjs',
+  name: 'HorizontalBarDAOChartView',
+  extends: 'org.chartjs.AbstractChartView',
+
+  implements: [
+    'foam.mlang.Expressions'
+  ],
+
+  properties: [
+    {
+      class: 'Map',
+      name: 'config',
+      documentation: `
+        The config map that is expected by chartjs. Structure and information can be found in chartjs.org's documentation.
+      `,
+      factory: function () {
+        var self = this;
+        return {
+          type: 'horizontalBar',
+          options: {
+            legend: {
+              display: false
+            },
+            elements: {
+              rectangle: {
+                borderWidth: 2,
+              }
+            },
+            scales: {
+              yAxes: [{
+                ticks: {
+                  // convert to millions
+                  callback: function (value, index, values) {
+                    const dateArray = value.split('/');
+                    const monthNames = [
+                      "January", "February", "March", "April", "May", "June",
+                      "July", "August", "September", "October", "November", "December"
+                    ];
+                    const quarterNames = ['Q1', 'Q2', 'Q3', 'Q4'];
+
+                    switch (self.dateFrequency) {
+                      case net.nanopay.liquidity.ui.dashboard.DateFrequency.MONTHLY:
```

Comment:
This isn't part of the foam repo so it shouldn't be here. You can override the config wherever you're rendering this view. I'd keep the config of this model super simple:
```
{ type: 'horizontalBar' }
```

Reply:
Done.